crypto/fipsmodule/sha/sha3_test.cc

```diff
@@ -91,8 +91,61 @@ class SHA3TestVector {
 
   }
 
+  void NISTTestVectors_SHAKE128() const {
+    uint32_t digest_length = out_len_ / 8;
+    uint8_t *digest = new uint8_t[digest_length];
+
+    #if !defined(OPENSSL_ANDROID)
```

Comment:
didn't we get rid of these in the SHA3 tests? Can we do it here as well?

Reply:
I will keep the branch updated with main. Sorry for this.